client/dashboard/utils/site-features.ts

```diff
@@ -0,0 +1,48 @@
+import type { Site } from '../data/types';
+
+/**
+ * Centralized place for site feature checks
+ * This helps maintain consistency and makes it easier to update feature detection logic
+ */
+
+export const SiteFeatures = {
+	/**
+	 * Checks if a site is an Atomic site
+	 */
+	isAtomic: ( site: Site ): boolean => site.is_wpcom_atomic,
+
+	/**
+	 * Checks if a site is a staging site
+	 */
+	isStaging: ( site: Site ): boolean => site.is_wpcom_staging_site,
+
+	/**
```

Comment:
I believe all the comments here are useless because they are literally just reiterating the method names 🙈